.cbmc-batch/proof_helpers.h

```diff
@@ -1,32 +1,55 @@
 #include<stdlib.h>
 
+#define MAX_INITIAL_ITEM_ALLOCATION 2
+#define MAX_ITEM_SIZE 15
+#define MAX_STR_LEN 32
+#define MAX_BUF_LEN 32
+
 #define ASSUME_VALID_MEMORY(ptr) ptr = malloc(sizeof(*(ptr)))
 #define ASSUME_VALID_MEMORY_COUNT(ptr, count) ptr = malloc(sizeof(*(ptr)) * (count))
 #define ASSUME_DEFAULT_ALLOCATOR(allocator) allocator = aws_default_allocator()
 #define ASSUME_ARBITRARY_ARRAY_LIST(list, item_count, item_size) list = get_arbitrary_array_list( (item_count), (item_size) )
 
 int nondet_int();
 size_t nondet_size_t();
+void *nondet_void_ptr();
 
 struct aws_array_list *get_arbitrary_array_list(size_t item_count, size_t item_size) {
```

Comment:
`item_count` and `item_size` should be as non-deterministic as possible. So why should we assume in our proofs:

`__CPROVER_assume(item_count <= MAX_INITIAL_ITEM_ALLOCATION);`

`__CPROVER_assume(item_size <= MAX_ITEM_SIZE);`